src/Dfc.CourseDirectory.Core/DataManagement/ParsedCsvCourseRow.cs

```diff
@@ -52,9 +52,20 @@ public static ParsedCsvCourseRow FromCsvCourseRow(CsvCourseRow row, IEnumerable<
             _ => (CourseAttendancePattern?)null
         };
 
+        public static string ResolveAttendancePattern(CourseAttendancePattern value) => value switch
+        {
+            CourseAttendancePattern.Daytime => "daytime",
+            CourseAttendancePattern.Evening => "evening",
+            CourseAttendancePattern.Weekend => "weekend",
+            CourseAttendancePattern.DayOrBlockRelease => "day/block release",
+            _ => null
```

Comment:
I think the `_` case should throw
```suggestion
            _ => throw new NotSupportedException($"Unknown value: '{value}'.")
```

Reply:
changed